cloud_edition/flexibility_mode/docs/partner_starterkit.rst

```diff
@@ -36,6 +36,7 @@ We provide some aliases to do some `superuser tasks`_
 
 
 .. _`Partners Portal`: https://partners.akeneo.com
+.. _`Create a new cloud ticket to the support`: https://helpdesk.akeneo.com 
```

Comment:
Hello! if this is only for cloud, you can put this url: https://akeneo.atlassian.net/servicedesk/customer/portal/8/group/23 instead :)
Thanks!

Reply:
:ah: